commands/test.js

```diff
@@ -112,6 +111,7 @@ module.exports = {
       }
 
       log.debug("Testing artifacts");
+      await exec(`${shellDir}/common/initialize-dependencies-java.sh ${taskParams["languageVersion"]}`);  
```

Comment:
Just a bit confused w/ this. Why do we run it three separate times instead of the once initially?

Reply:
@timrbula I'm glad you asked ... :-)

The reason for the repeated calling is because the Java version gets implicitly set to v17 just prior to calling SonarQube in both the Unit and Static scripts ... so we reset the version back to the user-specified version before each test type so any compiles and/or test maven/gradle tasks use the correct Java version.

Reply:
got it! thanks for the explanation